service/policies.go

```diff
@@ -0,0 +1,71 @@
+package service
+
+import (
+	"reflect"
+
+	v1 "code.vegaprotocol.io/protos/vega/wallet/v1"
+	"github.com/golang/protobuf/jsonpb"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+type ConsentConfirmation struct {
+	TxStr    string
+	Decision bool
+}
+
+type ConsentRequest struct {
+	tx *v1.SubmitTransactionRequest
+}
+
+func (r *ConsentRequest) String() (string, error) {
+	data, err := protojson.Marshal(r.tx)
```

Comment:
Do we not want to print this with nice indentation? Currently it just prints a wall of text of json.